esmvaltool/preprocessor/_time_area.py

```diff
@@ -56,7 +56,10 @@ def extract_season(cube, season):
     season: str
         Season to extract. Available: DJF, MAM, JJA, SON
     """
-    iris.coord_categorisation.add_season(cube, 'time', name='clim_season')
+    try:
```

Comment:
Replace the try except block by a check if the season coordinate is already available:
```
if not cube.coord('clim_season'):
    iris.coord_categorisation.add_season(cube, 'time', name='clim_season')
```
because this try/except may silently discard many other errors.